lib/rubocop/cop/rspec/capybara/specific_matcher.rb

```diff
@@ -71,11 +71,16 @@ class SpecificMatcher < Base
             (send nil? _ (str $_) ... )
           PATTERN
 
+          # @!method options(node)
+          def_node_search :options, <<-PATTERN
+            (pair (sym $_) _)
+          PATTERN
```

Comment:
You can turn this into
```suggestion
          def_node_search :option?, <<-PATTERN
            (pair (sym %) _)
          PATTERN
```
And then use 
```
option?(node, :href) || attributes.include?('href')
```

Reply:
That looks very much better. Thank you!